Sources/LiveViewNative/Live/LiveErrorView.swift

```diff
@@ -29,7 +29,7 @@ public struct LiveErrorView<Fallback: View>: View {
            case let .ConnectionError(trace) = error
         {
             SwiftUI.VStack {
-                WebErrorView(html: trace)
+                WebErrorView(html: trace.errorText)
```

Comment:
This error now contains a `LiveChannel?` , if `.some` the channel must call `shutdownParentSocket` in order to free it's connection, it also must listen for `assets_change` events